lib/base/initialize.cpp

```diff
@@ -5,7 +5,7 @@
 
 using namespace icinga;
 
-bool icinga::InitializeOnceHelper(void (*func)(), int priority)
+bool icinga::InitializeOnceHelper(const std::function<void()>& func, InitializePriority priority)
```

Comment:
IMAO should be moved.

Reply:
`Loader::AddDeferredInitializer()` already takes a const reference. For a move to do anything useful here, you'd have to change more places (totally doable, but that's why I did it like it is here).